modules/thermo-cycler/thermo-cycler-arduino/thermo-cycler-arduino.ino

```diff
@@ -242,6 +243,17 @@ void update_fans_from_state()
 
   if (millis() - last_checked > 100)
   {
+    if (!auto_fan)
+    { // Heatsink safety threshold overrides manual operation
+      if (temp_probes.heat_sink_temperature() > 55)
+      {
+        // Fan speed proportional to temperature
+        float pwr = temp_probes.heat_sink_temperature() / 100;
```

Comment:
Worried about types here and on 283, maybe make this a 100.0? Also maybe have a HEATSINK_P_CONSTANT defined somewhere that can be 1.0 for now?

Reply:
Yeah in general there's some random thermal cutoffs especially in the fan stuff that I'd love to see as #defines somewhere.

Reply:
heat_sink_temperature() is a float so we're actually good here but yea, I see how it's easy to have a bad type conversion. I'll make the changes